tests/helpers/test_condition.py

```diff
@@ -66,3 +67,12 @@ def test_or_condition(self):
 
         self.hass.states.set('sensor.temperature', 100)
         assert test(self.hass)
+
+    def test_time_window(self):
```

Comment:
it shouldn't matter because we simply test that now is either between 6am and 6pm or between 6pm and 6am.

I guess it could fail when we test as the second ticks over from 06:00:00 to 06:00:01 between the two tests, but I'm honestly not that worried about that because the window is much smaller than a second.